### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://hosted.weblate.org/projects/tomb/tomb/) for Tomb/tomb.

Current translation status:

Weblate translation status

### CODE REVIEW

File: extras/translations/de.po [hunk 1]
```diff
@@ -531,7 +535,9 @@ msgstr "Verschlüsselungsschlüssel konnte nicht erstellt werden."
 
 #: tomb:Create:forge_key:1492
 msgid "Commanded to forge key ::1 key:: with cipher algorithm ::2 algorithm::"
-msgstr "Der Schlüssel ::1 key:: wird mit dem Verschlüsselungsalgorithmus ::2 algorithm:: geschmiedet."
+msgstr ""
+"Der Schlüssel ::1 key:: wird mit dem Verschlüsselungsalgorithmus ::2 "
+"algorithm:: gefälscht"
```
Comment (Narrat) [inline extras/translations/de.po:540]: Well, this isn't a wrong translation for forge, but it is wrong in this case

Comment (jaromil) [reply extras/translations/de.po:540]: what you mean? the swedish patch seems fine

Comment (Narrat) [reply extras/translations/de.po:540]: In this case forge was translated as to fake something. Not in its meaning to create something. And this particular german word doesn't have multiple meanings.
Translated back it kinda sounds like: the key is faked with the algorithm. Not really convincing to read :P

**Comment by jaromil** [comment]
Many thanks for the contribution!

**Pull request merged**